docker-compose.yml

```diff
@@ -7,10 +7,20 @@ web:
   ports:
     - "8282:8282"
   volumes:
-    - app:/mnt/app
+    - app:/mnt/app   # DO NOT USE IN PRODUCTION, FOR LOCAL DEV ONLY!
 db:
```

Comment:
quick note: this docker-compose.yml file is pretty much production ready -except- for this line, which is used by developers only.  we may need to do something in the future like keep this file as a template, and it's built along with the containers.  Based on whether or not you are trying to expose the code, or just run this thing in production, we could keep docker-compose.yml as a template and add or remove this line dynamically.  or something.  need to think about this carefully.